fhir-install/Dockerfile

```diff
@@ -36,15 +36,16 @@ ENV WLP_OUTPUT_DIR=/opt/ol/wlp/usr/servers
 # Tell liberty not to worry about a keystore since we provide our own at a different path
 ENV KEYSTORE_REQUIRED "false"
 
-# Replace links to defaultServer with links to fhir-server
+# Replace links to defaultServer with links to fhir-server and 
+# create the configDropins directories for the fhir-server
 USER root
 RUN ln -sfn ${FHIR} /output && \
     ln -sfn ${FHIR} /config && \
     rm -rf /opt/ol/wlp/output && \
-    rm -rf /opt/ol/wlp/usr/servers/defaultServer
+    rm -rf /opt/ol/wlp/usr/servers/defaultServer && \
+    mkdir -p /config/configDropins/defaults /config/configDropins/overrides && \
```

Comment:
I prefer this one to be down below because I prefer having this directory owned by the default user (and not root).  I don't think the extra layer is a big deal.